src/layouts/AppHeader.js

```diff
@@ -94,12 +94,21 @@ class AppHeader extends React.Component {
                   }
                   style={{ float: "right" }}
                 >
-                  <Menu.Item key="switchLang" onClick={switchLang}>
-                    <FormattedMessage
-                      id='lang'
-                      defaultMessage='Language:English'
-                    />
-                  </Menu.Item>
+                  {[{id: "zh-Hans", name: "简体中文"},
```

Comment:
Since this button act differently based on the login state of user.  It's better to move it to the `if (isLoggedIn)` part above.